test/segment-transmuxer.test.js

```diff
@@ -53,6 +54,7 @@ QUnit.module('Segment Transmuxer', {
     assert.timeout(5000);
   },
   afterEach(assert) {
+    dispose();
```

Comment:
Not sure how this didn't come up earlier, maybe qunit is  faster now. Basically we have to cleanup the segment-transmuxer queue after each test.